pkg/vm/createEvm.go

```diff
@@ -6,42 +6,64 @@ import (
 	"bytes"
 	"encoding/json"
 	"errors"
-	"fmt"
 	"math/big"
 
-	"github.com/ava-labs/avalanche-cli/cmd/prompts"
 	"github.com/ava-labs/avalanche-cli/ux"
 	"github.com/ava-labs/avalanchego/utils/logging"
 	"github.com/ava-labs/subnet-evm/core"
 	"github.com/ava-labs/subnet-evm/params"
 )
 
-func CreateEvmGenesis(name string, log logging.Logger) ([]byte, error) {
+func nextStage(currentState wizardState, direction stateDirection) wizardState {
+	switch direction {
+	case forward:
+		currentState += 1
+	case backward:
+		currentState -= 1
+	default:
+		return errored
+	}
+	return currentState
+}
+
+func CreateEvmGenesis(name string, log logging.Logger) ([]byte, string, error) {
 	ux.Logger.PrintToUser("creating subnet %s", name)
 
 	genesis := core.Genesis{}
 	conf := params.SubnetEVMDefaultChainConfig
 
-	chainId, err := getChainId()
-	if err != nil {
-		return []byte{}, err
-	}
-	conf.ChainID = chainId
+	stage := startStage
 
-	*conf, err = getFeeConfig(*conf)
-	if err != nil {
-		return []byte{}, err
-	}
+	var (
+		chainId    *big.Int
+		tokenName  string
+		allocation core.GenesisAlloc
+		direction  stateDirection
+		err        error
+	)
 
-	allocation, err := getAllocation()
-	if err != nil {
-		return []byte{}, err
+	for stage != doneStage {
```

Comment:
I shall put the stages definitions on this file, as it is the only one using them